cobalt/testing/browser_tests/BUILD.gn

```diff
@@ -346,7 +351,7 @@ test("cobalt_browsertests") {
   ]
 
   data = [
-    "data/",
+    "//cobalt/testing/browser_tests/data/",
```

Comment:
nit: Could we depend on //content/shell//browser_tests/data ?
That way we wouldn't need to copy those files, and ultimately
the dependency would be for testing only.